### PR TITLE
[4.0] Selected language in Backend Login has no effect (Ref #26148)

### PR DESCRIPTION
**Pull request by bees4ever**
Pull Request for Issue #26148.

### Summary of Changes
The login module posts now also section values if the value is not empty.

### Testing Instructions
1. Apply the patch
1. Rebuild the JavaScript (`npm run build:js`)
1. Install at least your a second language, i.e. "German"
1. Log out from the backend and open the backend login mask
1. Select another langauge as the default one, i.e. "German"
1. The backend should now be translated correctly. 
1. Log out and in again with the default language. The backend should be default translated.

### Expected result
If I selected a langauge in the login mask, this language should be the backend language for this login.

### Actual result
Change of language has no effect.

### Documentation Changes Required
As I can see, no. But I am not sure.

### CODE REVIEW

File: build/media_source/mod_login/js/admin-login.es6.js [hunk 1]
```diff
@@ -39,6 +39,8 @@
 
           if (element.hasAttribute('name') && element.nodeName === 'INPUT') {
             segments.push(`${encodeURIComponent(element.name)}=${encodeURIComponent(element.value)}`);
+          } else if (element.hasAttribute('name') && (element.nodeName === 'SELECT') && element.value.length > 0) {
```
Comment (Quy) [inline build/media_source/mod_login/js/admin-login.es6.js]:
```suggestion
          } else if (element.hasAttribute('name') && element.nodeName === 'SELECT' && element.value.length > 0) {
```

**Comment by roland-d** [comment]
I have tested this item :white_check_mark: successfully on a599c9911c6a30471aa12e34ec09cc7c44670762<br /><br />First I logged in with French but the language is still English, then I applied the patch and logged in with French again, now the backend is in French as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26150">issues.joomla.org/tracker/joomla-cms/26150</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by sebenns** [comment]
I have tested this item :white_check_mark: successfully on a599c9911c6a30471aa12e34ec09cc7c44670762<br /><br />First I tested the issue, then applied the patch and relogged in with German. The backend has been translated in German as expected. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26150">issues.joomla.org/tracker/joomla-cms/26150</a>.</sub>

**Pull request merged**